app/api/admin.py

```diff
@@ -1,31 +1,49 @@
 from flask import jsonify, Blueprint
 from app.token import admin_required
-from app.dob import (add_user, user_task, task_delete, update)
+from app.dob import (add_user, user_task, task_delete, update, orgnization)
 from app.schema import (TaskSchema, InfoSchema,
-                        UserSchema, UpdateSchema)
-from app.util import data_now_json_str
+                        UserSchema, UpdateSchema, OrgnizationSchema)
+from app.util import data_now_json_str, role_valid
 admin_bp = Blueprint('admin', __name__, url_prefix='/admin')
 
 
+@admin_bp.route('/create_orgnization', endpoint='create_orgnization', methods=['POST'])
+def create_orgnization():
```

Comment:
Need to change this, because like that anyone can create an admin role and organization.